paper-dropdown-menu-light.html

```diff
@@ -0,0 +1,562 @@
+        transition-timing-function: cubic-bezier(.4,0,.2,1);
+        color: var(--paper-dropdown-menu-color, --secondary-text-color);
+        @apply(--paper-dropdown-menu-label);
+      }
+
+      :host([no-label-float]) label {
+        top: 8px;
+      }
+
+      label.label-is-floating {
+        font-size: 12px;
+        top: 8px;
+      }
+
+      label.label-is-hidden {
+        display: none;
+      }
+
+      :host([focused]) label.label-is-floating {
+        color: var(--paper-dropdown-menu-focus-color, --primary-color);
+      }
+
+      :host([invalid]) label.label-is-floating {
+        color: var(--paper-dropdown-error-color, --error-color);
+      }
+
+      /**
+       * Sets up the focused underline. It's initially hidden, and becomes
+       * visible when it's focused.
+       */
+      label:after {
+        background-color: var(--paper-dropdown-menu-focus-color, --primary-color);
+        bottom: 8px;    /* The container has an 8px bottom padding */
+        content: '';
+        height: 2px;
+        left: 45%;
+        position: absolute;
+        transition-duration: .2s;
+        transition-timing-function: cubic-bezier(.4,0,.2,1);
+        visibility: hidden;
+        width: 8px;
+        z-index: 10;
+      }
+
+      :host([invalid]) label:after {
+        background-color: var(--paper-dropdown-error-color, --error-color);
+      }
+
+      :host([no-label-float]) label:after {
+        bottom: 8px;    /* The container has a 8px bottom padding */
+      }
+
+      :host([focused]:not([disabled])) label:after {
+        left: 0;
+        visibility: visible;
+        width: 100%;
+      }
+
+      iron-icon {
+        position: absolute;
+        right: 0px;
+        bottom: 8px;    /* The container has an 8px bottom padding */
+        @apply(--paper-font-subhead);
+        margin-top: 12px;
+        color: var(--disabled-text-color);
+        @apply(--paper-dropdown-menu-icon);
+      }
+
+      :host([no-label-float]) iron-icon {
+        margin-top: 0px;
+      }
+
+      .error {
+        display: inline-block;
+        visibility: hidden;
+        color: var(--paper-dropdown-error-color, --error-color);
+        @apply(--paper-font-caption);
+        position: absolute;
+        left:0;
+        right:0;
+        bottom: -12px;
+      }
+
+      :host([invalid]) .error {
+        visibility: visible;
+      }
+    </style>
+
+    <!-- this div fulfills an a11y requirement for combobox, do not remove -->
+    <div role="button"></div>
+    <paper-menu-button
+      id="menuButton"
+      vertical-align="[[verticalAlign]]"
+      horizontal-align="[[horizontalAlign]]"
+      vertical-offset="[[_computeMenuVerticalOffset(noLabelFloat)]]"
+      disabled="[[disabled]]"
+      no-animations="[[noAnimations]]"
+      on-iron-select="_onIronSelect"
+      on-iron-deselect="_onIronDeselect"
+      opened="{{opened}}">
+      <div class="dropdown-trigger">
+        <div class="container">
```

Comment:
`container` is holding the absolutely positioned `label`, which does `top` math, so I am worried about removing it.